src/makepkg/functions/lint_pkgbuild/pkgname.sh

```diff
@@ -73,6 +73,16 @@ lint_one_pkgname() {
 		ret=1
 	fi
 
+	if [[ "${pkgname}" =~ [A-Z] ]]; then
+		error "'pkgname' contains capital letters"
+		ret=1
+	fi
+
+	if [[ "${pkgbase}" =~ [A-Z] ]]; then
+  	error "'pkgbase' contains capital letters"
+  	return 1
+	fi
+
```

Comment:
This should be the last thing, but you could be able to put these under one `if` statement but just checking the `${type}` and `${name}` variables as with the statements above it.